server/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "hp-admin-crypto-server"
+version = "0.1.0"
+authors = ["PJ <pj@imagine-nyc.com>"]
+edition = "2018"
+
+[dependencies]
+hpos-state-core = { git = "https://github.com/Holo-Host/hpos-state", branch = "develop" }
```

Comment:
We won't be able to build this in Nix using this kind of incantation, right, @yegortimoshenko?

Reply:
I as able to make this work past the rust build part, but still other build issues with this PR


https://github.com/Holo-Host/hp-admin-crypto/pull/3

I will work on it more today and keep my work in the https://github.com/Holo-Host/hp-admin-crypto/pull/3

Reply:
Great, thanks!